front/components/assistant/AssistantBrowser.tsx

```diff
@@ -82,20 +82,17 @@ export function AssistantBrowser({
       list: filteredAgents.filter(
         (a) => a.scope === "published" && a.userListStatus === "in-list"
       ),
-      // TODO: Implement most popular agents (upcoming PR for issue #5454)
       most_popular: filteredAgents
         .filter((a) => a.usage && a.usage.messageCount > 0)
         .sort(
           (a, b) => (b.usage?.messageCount || 0) - (a.usage?.messageCount || 0)
-        )
-        .slice(0, 0), // Placeholder -- most popular agents are not implemented yet
+        ),
```

Comment:
Goal would be to have most popular shown as default tab -- but with fast loading
Previous code handled that for company by only grabbing 24 agents
How is it done now?

Reply:
This is how it's supposed to be done here: we first query the 24 most popular agents and then retrieve their author.
Most popular is indeed the default tab 😃

Reply:
What happens if there is no usage yet? Do we show an empty tab? Or do properly skip all empty tabs?

Reply:
Empty tabs are skipped 😃

Reply:
Could have been an issue since in that case we do the full agent call that can be slow
But if there's no usage, then there's not many agents, so that's actually fine